Migrate away from gnome-common deprecated vars and macros

maxyz opened the pull request:
gnome-common is deprecating some vars and macros, listed here:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=829932

This change follows the gnome recommendations from:
https://wiki.gnome.org/Projects/GnomeCommon/Migration

itzexor commented on configure.ac:
```diff
@@ -363,13 +351,12 @@ echo "
         XSync support:            ${have_xsync}
         XTest support:            ${have_xtest}
-        Legacy UPower backend:    ${have_old_upower}
+	Legacy UPower backend:    ${have_old_upower}
```
You replaced the spaces with a tab here.

maxyz commented:
¡Hola itzexor!

El 2016-09-17 a las 15:04 -0700, itzexor escribió:
> @itzexor commented on this pull request.
>
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
>
> In configure.ac:
>
>>          IPv6 support:             ${have_full_ipv6} 
>         Backtrace support:        ${have_backtrace} 
>         XRender support:          ${have_xrender} 
>         XSync support:            ${have_xsync} 
>         XTest support:            ${have_xtest} 
> -        Legacy UPower backend:    ${have_old_upower} 
> +       Legacy UPower backend:    ${have_old_upower}
>
> You replaced the spaces with a tab here.

Something went wrong with the rebase and the qt5.7 change. I fixed that, and 
also changed the tabs by spaces in this file.

Happy hacking,
-- 
"The use of COBOL cripples the mind; its teaching should, therefore, be
regarded as a criminal offense."
-- Edsger W. Dijkstra
 Saludos /\/\ /\ >< `/

clefebvre commented:
Hi @maxyz , you're changing the version in configure.ac.. [3.0.0]

maxyz commented:
Thanks for noticing it. Fixed.
